composer.json

```diff
@@ -44,15 +44,15 @@
         "php": "^7.4 || ^8.0",
         "ext-phar": "*",
         "composer-plugin-api": "^2.2",
-        "amphp/parallel-functions": "^1.0",
+        "amphp/parallel-functions": "dev-master",
```

Comment:
Although this good for running the tests I would prefer to wait for the release before merging the PR

Reply:
Updated to `^1.1` now 🥳